src/components/formsCommons/WithSwal.js

```diff
@@ -17,16 +16,19 @@ export const withSwal = ({
     })
     Swal.showLoading()
     promise()
-        .then(result => {
-            if (!result.ok) throw Error(resultWasntOkErrorText || "Something went wrong!")
+        .then(response => response.json().then(data => ({status: response.status, result: data})))
+        .then(({status, result}) => {
+            if (!(Math.floor(status / 100) === 2)) {
```

Comment:
To trochę wygląda jak magic numbers

Reply:
Akurat to jest sprawdzenie, czy status to 2xx XDDD ale mogę zmienić na jakąś funkcję pomocniczą, why not XDD

Reply:
Znaczy ja wiem co to robi ale wygląda jak magic numbes XD